### PR TITLE
not core

### PR DESCRIPTION
**Pull request by brianteeman**
Pull Request for Issue # .
image

Updated strings - they are not core files etc

Note from the screenshot we can see that the _1 string was not used when it should have been

### CODE REVIEW

**Comment by Anu1601CS** [comment]
@brianteeman I am still confused here.
Because in update the core file of the overridden file is changed or update. So, this is fine ?

**Comment by brianteeman** [comment]
Core = joomla
Core != Extension

**Comment by brianteeman** [comment]
Maybe it should be

Files that were overridden have changed

**Comment by Anu1601CS** [comment]
> Core = joomla
Core != Extension

Ok, now I understand. Like `com_media` or `com_content` etc is the extension of Joomla but not the core of Joomla right?

**Comment by brianteeman** [comment]
no. Core is anything that is shipped with a default install of joomla
com_content = core
com_helloworld = extension

**Comment by Anu1601CS** [comment]
> no. Core is anything that is shipped with a default install of joomla
com_content = core
com_helloworld = extension

**Comment by Anu1601CS** [comment]
Because the user may create an override of default Joomla extensions like `com_content` and non-core like `com_helloworld`

**Comment by brianteeman** [comment]
yes - and on this screen we are only talking about non-core extensions

File: administrator/language/en-GB/en-GB.plg_installer_override.ini [hunk 1]
```diff
@@ -5,5 +5,5 @@
 
 PLG_INSTALLER_OVERRIDE="Installer - Override"
 PLG_INSTALLER_OVERRIDE_PLUGIN_XML_DESCRIPTION="The system overrides plugin enables notifications and handling of overrides after an update in case of differences."
-PLG_INSTALLER_N_OVERRIDE_FILE_UPDATED="%s Overrides core file updated."
-PLG_INSTALLER_N_OVERRIDE_FILE_UPDATED_1="Override core file updated."
+PLG_INSTALLER_N_OVERRIDE_FILE_UPDATED="%s Overriden files have changed."
```
Comment (Anu1601CS) [inline administrator/language/en-GB/en-GB.plg_installer_override.ini]: @brianteeman `Overridden` ?

**Comment by brianteeman** [comment]
fixed it

**Comment by Anu1601CS** [comment]
Thank you very much.

**Pull request merged**